src/components/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import Navbar from '../components/Navbar'
+import AllBeers from '../assets/beers.png'
+import NewBeers from '../assets/new-beer.png'
+import RandomBeer from '../assets/random-beer.png'
+import { Link } from 'react-router-dom'
+import axios from 'axios'
+
+
+function Home() {
+  const [oneBeer, setOneBeer] = useState('');
+  const handleOneBeer = (e) => setOneBeer(e.target.value);
+
+  const handleSearch = async (e) => {
+    e.preventDefault();
+
+    try {
+      await axios.get('https://ih-beers-api2.herokuapp.com/beers');
+
+    } catch (error) {
+      console.log(error)
+
+    }
+  }
+
+  return (
+    <div>
+      <Navbar />
+      <img src={AllBeers} alt="All Beers" />
+      <Link to='/beers' onClick={handleOneBeer} >
+        <h1>All Beers</h1>
+        <p>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</p>
+      </Link>
+      <img src={NewBeers} alt="New Beers" />
+      <Link to='/beers'>
```

Comment:
You created the right path on `App.js` but forgot to change them here. So accordingly to what you have on `App.js` here, the correct path for `New Beers` is `'/beers/NewBeer'`